src/haz3lcore/lang/Form.re

```diff
@@ -141,6 +141,8 @@ let forms: list((string, t)) = [
   ("plus", mk_infix("+", Exp, P.plus)),
   ("minus", mk_infix("-", Exp, P.plus)),
   ("times", mk_infix("*", Exp, P.mult)),
+  ("power_", mk_nul_infix("**", P.power)),
```

Comment:
Why are both `**` and `**.` mapped to the same operation? Seems we should have an integer and float version.

Reply:
Added integer power operator.